backend/core/src/listings/dto/listings-query-params.ts

```diff
@@ -47,24 +49,30 @@ export class ListingsQueryParams extends PaginationAllowsAllQueryParams {
   @ApiProperty({
     name: "orderBy",
     required: false,
-    enum: OrderByFieldsEnum,
     enumName: "OrderByFieldsEnum",
-    example: "updatedAt",
+    example: '["updatedAt"]',
+    isArray: true,
   })
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
-  @IsEnum(OrderByFieldsEnum, { groups: [ValidationsGroupsEnum.default] })
-  orderBy?: OrderByFieldsEnum
+  @IsArray({ groups: [ValidationsGroupsEnum.default] })
+  @ArrayMaxSize(16, { groups: [ValidationsGroupsEnum.default] })
+  @Type(() => ListingFilterParams)
+  @IsEnum(OrderByFieldsEnum, { groups: [ValidationsGroupsEnum.default], each: true })
+  @Validate(OrderQueryParamValidator, { groups: [ValidationsGroupsEnum.default] })
+  orderBy?: OrderByFieldsEnum[]
 
   @Expose()
   @ApiProperty({
     enum: OrderParam,
-    example: "DESC",
-    default: "DESC",
+    example: '["DESC"]',
+    default: '["DESC"]',
     required: false,
+    isArray: true,
   })
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
-  @IsString({ groups: [ValidationsGroupsEnum.default] })
-  @IsIn(Object.keys(OrderParam), { groups: [ValidationsGroupsEnum.default] })
-  @Transform((value: string | undefined) => (value ? value : OrderParam.DESC))
```

Comment:
since Transform is no longer needed in this file, you can remove it from the import.